holoviews/core/util.py

```diff
@@ -2031,7 +2031,7 @@ def is_nan(x):
     Checks whether value is NaN on arbitrary types
     """
     try:
-        return np.isnan(x)
+        return bool(pd.isna(x))
```

Comment:
```suggestion
        # Using pd.isna instead of np.isnan as np.isnan(pd.NA) returns pd.NA!
        # Call bool() to raise an error if x is pd.NA, an array, etc.
        return bool(pd.isna(x))
```

Reply:
Thank you :+1: